package.json

```diff
@@ -137,5 +140,68 @@
     "tslint": "~6.1.0",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "~4.0.3"
+  },
+  "bit": {
```

Comment:
please delete all the bit code configurations
This was probably standing on a very old branch.